### ISSUE TITLE
Feature req - notify when bitcoind is down

### ISSUE THREAD

**Issue by ephraim71** [ElementsProject/lightning#808]
Something like Eclair's "connection lost" message.    
Right now it behaves as if nothing happened when I close bitcoind

**Comment by jb55** [comment]
related: #1652

**Comment by rustyrussell** [comment]
Yes, we now time out and eventually stop if bitcoind goes non-reponsive.

**Comment by Sjors** [comment]
Bitcoin Core does have a few other things you might be able to use for detecting shutdown:

- the presence of `bitcoind.pid` (if you know where it is and have access)
- the `shutdownnotify` setting, e.g. `-shutdownnotify='systemd-notify --stopping` (but this requires the user to configure it)
